src/components/LoanCards/LoanCardImage.vue

```diff
@@ -71,6 +71,10 @@ export default {
 		loanImageHash: {
 			type: String,
 			default: ''
+		},
+		useDefaultStyles: {
```

Comment:
Default styles can now be turned off with this prop. No changes were made to the style definitions.